src/modal/parts/Overlay.jsx

```diff
@@ -36,6 +62,7 @@ const Overlay = ({ contentMaxWidth }) => {
                 onClick={() => handleClose('Modal Overlay')}
                 style={{ right: 0, width: `calc((100% - ${contentMaxWidth}px) / 2)` }}
             />
+            {country === 'GB' ? vertOverlay() : null}
```

Comment:
Consider making this a little more generic. We should be able to use whether or not `contentMaxHeight` exists instead of relying on the country. We could also do the same thing for the exisiting horizontal overlay. We could do something like `{contentMaxWidth && <Fragment />}` and `{contentMaxHeight && <Fragment />}`.